### DISCUSSION TITLE
Welcome to mkdocs-material Discussions!

### DISCUSSION THREAD

**Discussion by squidfunk** [squidfunk/mkdocs-material#2096] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by jeffreryan** [comment]
mkdocs web frontend is with out frustration, the sexiest home page I have ever seen. It is going so forward that the mkdocs will be supported by all you AND me. I am just getting back into app dev for open app educational platforms. It's been a while I last coded in 1985! Thanks for using this collaboration platform to show me the light.

**Comment by jeffreryan** [comment]
I will get right on that thank uou
